lib/ramble/ramble/variants.py

```diff
@@ -255,6 +255,13 @@ def __init__(
         self.default = default
         self.description = description
         self.values = values
+        if isinstance(self.default, bool):
+            if self.default:
+                self._definition = f"+{self.name}"
+            else:
+                self._definition = f"~{self.name}"
```

Comment:
nit: do you think this would be cleaner more like:

```
       prefix = "+" if self.default else "~"
       self._definition = f"{prefix}{self.name}"
```

Reply:
Updated!